app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,20 @@
+class RelationshipsController < ApplicationController
+  before_action :require_user
+
+  def index
+
+  end
+
+  def create
+    user = User.find(params[:relation_id])
```

Comment:
you aren't using this `user` variable anywhere it appears - if you aren't you can delete it.